quirks/change-password-URLs.json

```diff
@@ -17,8 +21,10 @@
     "gmx.net": "https://account.gmx.net/ciss/security/edit/passwordChange",
     "gov.br": "https://acesso.gov.br/area-cidadao/#/alterarSenha",
     "impots.gouv.fr": "https://cfspart.impots.gouv.fr/monprofil-webapp/GererMonProfil",
+    "instagram.com": "https://www.instagram.com/accounts/password/change/",
```

Comment:
Good URL. It’s kind of surprising, given that facebook.com has a solid adoption of the Well-Known URL for Changing Passwords.

Reply:
:facebook: